Core/FileStore.swift

```diff
@@ -22,16 +22,30 @@ import Configuration
 
 public class FileStore {
 
-    private let groupIdentifier: String = ContentBlockerStoreConstants.groupName
+    private let groupIdentifier: String = ContentBlockerStoreConstants.configurationGroupName
```

Comment:
Since this is used for the file paths: is there anything that will get reset here that might cause trouble for users?